src/libs/suggest.js

```diff
@@ -44,9 +44,6 @@ export const selectItem = (selectedItem, { query, replaceUrl = false, fromQueryP
 };
 
 export const getInputValue = item => {
-  if (item instanceof Poi) {
```

Comment:
Technically this check `item instanceof Poi` could be preserved. However it makes testing more cumbersome as some tests trigger simulated events using serialized POIs. And avoid using `instanceof` is probably preferable.